diff --git a/node_modules/electron-updater/out/LinuxUpdater.js b/node_modules/electron-updater/out/LinuxUpdater.js
index a7587ae..6a84390 100644
--- a/node_modules/electron-updater/out/LinuxUpdater.js
+++ b/node_modules/electron-updater/out/LinuxUpdater.js
@@ -34,7 +34,7 @@ class LinuxUpdater extends BaseUpdater_1.BaseUpdater {
         if (/pkexec/i.test(sudo[0]) || sudo[0] === "sudo") {
             wrapper = "";
         }
-        return this.spawnSyncLog(sudo[0], [...(sudo.length > 1 ? sudo.slice(1) : []), `${wrapper}/bin/bash`, "-c", `'${commandWithArgs.join(" ")}'${wrapper}`]);
+        return this.spawnSyncLog(sudo[0], [...(sudo.length > 1 ? sudo.slice(1) : []), `/bin/bash`, "-c", `"${commandWithArgs.join(" ")}"`]);
     }
     sudoWithArgs(installComment) {
         const sudo = this.determineSudoCommand();
